akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -1707,7 +1707,9 @@ final class Flow[In, Out, Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends Gr
    * streams, one for each element key. The key is computed for each element
    * using the given function. When a new key is encountered for the first time
    * a new substream is opened and subsequently fed with all elements belonging to
-   * that key.
+   * that key. If `allowClosedSubstreamRecreation` is set to `true` a substream
```

Comment:
Good text, though make it an `@param allowClosedSubstreamRecreation` in a new line